### PR TITLE
Updates to task-based evaluation.

### PR DESCRIPTION
**Pull request by TheRootOf3**
Added new tasks to eval framework, added raw model evaluation script and scheduler submission scripts.

### CODE REVIEW

**Comment by TheRootOf3** [comment]
image

8tcvek

File: eval_framework_tasks/eval_results.py
Comment (Adamliu1) [inline eval_framework_tasks/eval_results.py:1]: why we don't use acc_norm any more?

File: eval_framework_tasks/experiment_data/experiments1/Gemma-eval1/wandb/wandb/debug-internal.log
Comment (Adamliu1) [inline eval_framework_tasks/experiment_data/experiments1/Gemma-eval1/wandb/wandb/debug-internal.log:1]: do we need these??????

File: eval_framework_tasks/experiment_data/experiments1/Aya23-eval1/wandb/wandb/offline-run-20240601_174058-inopzlhc/files/wandb-metadata.json
Comment (Adamliu1) [inline eval_framework_tasks/experiment_data/experiments1/Aya23-eval1/wandb/wandb/offline-run-20240601_174058-inopzlhc/files/wandb-metadata.json:1]: I dont like these wandb stuff... do we have to have it here?

**Review by Adamliu1 — APPROVED** [review]
LGTM but do we need wandb stuff? like log, it's quite a lot. However, I still approved this pr as they are just experiment results, they all look fine to  me

File: eval_framework_tasks/eval_results.py
Comment (TheRootOf3) [reply eval_framework_tasks/eval_results.py:1]: dropped for consistency.

File: eval_framework_tasks/experiment_data/experiments1/Gemma-eval1/wandb/wandb/debug-internal.log
Comment (TheRootOf3) [reply eval_framework_tasks/experiment_data/experiments1/Gemma-eval1/wandb/wandb/debug-internal.log:1]: probs not, added automatically.

File: eval_framework_tasks/experiment_data/experiments1/Aya23-eval1/wandb/wandb/offline-run-20240601_174058-inopzlhc/files/wandb-metadata.json
Comment (TheRootOf3) [reply eval_framework_tasks/experiment_data/experiments1/Aya23-eval1/wandb/wandb/offline-run-20240601_174058-inopzlhc/files/wandb-metadata.json:1]: probs not, added automatically.

**Pull request merged**